index.bs

```diff
@@ -1039,11 +1037,11 @@ RemoteReference = {
 Issue: handle "stale object reference" case.
 
 <div algorithm>
-To <dfn>deserialize remote reference</dfn> given a |remote reference|:
+To <dfn>deserialize remote reference</dfn> given a |remote reference| in |realm|:
```

Comment:
This is probably fine, but I've typically used a less prose, more function call, style where this would be written `To <dfn>deserialize remote reference</dfn> given |realm|, and |remote reference|`.

Reply:
done